### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by vladimirpetrov**
Capitalisation of known object/resource types: Templates, Network, Snapshot, User, Account, fixing many typos and grammar mistakes.

### CODE REVIEW

**Comment by borisstoyanov** [comment]
@blueorangutan docbuild

**Comment by vladimirpetrov** [comment]
@blueorangutan docbuild

File: source/adminguide/networking/inter_vlan_routing.rst [hunk 1]
```diff
@@ -22,36 +22,36 @@ traffic between VLANs. This feature enables you to build Virtual Private
 Clouds (VPC), an isolated segment of your cloud, that can hold
 multi-tier applications. These tiers are deployed on different VLANs
 that can communicate with each other. You provision VLANs to the tiers
-your create, and VMs can be deployed on different tiers. The VLANs are
+your create, and instances can be deployed on different tiers. The VLANs are
```
Comment (andrijapanicsb) [inline source/adminguide/networking/inter_vlan_routing.rst:25]: Capitalise "instances" ?

File: source/adminguide/networking/ip_forwarding_and_firewalling.rst [hunk 2]
```diff
@@ -248,11 +248,11 @@ To set up port forwarding:
    zone in CloudStack. See Adding a Zone and Pod in the Installation
    Guide.
 
-#. Add one or more VM instances to CloudStack.
+#. Add one or more instances to CloudStack.
 
 #. In the left navigation bar, click Network.
 
-#. Click the name of the guest network where the VMs are running.
+#. Click the name of the guest network where the instances are running.
```
Comment (andrijapanicsb) [inline source/adminguide/networking/ip_forwarding_and_firewalling.rst:255]: Should this whole file be checked for the capital "Instance" vs "instance" ?

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@vladimirpetrov pls recheck some comments left and resolve conflicts when possible please.

**Comment by vladimirpetrov** [comment]
@blueorangutan docbuild

**Comment by vladimirpetrov** [comment]
@blueorangutan docbuild

**Comment by vladimirpetrov** [comment]
@blueorangutan docbuild

**Review by andrijapanicsb — APPROVED** [review]
LGTM

**Comment by vladimirpetrov** [comment]
@blueorangutan docbuild

**Review by shwstppr — APPROVED** [review]
LGTM

**Pull request merged**